time.js

```diff
@@ -0,0 +1,14 @@
+'use strict'
+function durationBetweenDates (initalDateStr = "14 Jan 1992", endDateStr = "11 March 2023", timeUnit = "days") {

```

Comment:
What is returning if someone calls this function with the wrong timeUnit?

Reply:
It would throw NaN + timeUnit value. Handled it with check and warning massage.